bundles/persistence/org.openhab.persistence.jdbc/java/org/openhab/persistence/jdbc/db/JdbcBaseDAO.java

```diff
@@ -77,6 +87,42 @@ public JdbcBaseDAO() {
         initSqlQueries();
     }
 
+    /**
+     * ## Get high precision by fractal seconds, examples ##
+     *
+     * mysql > 5.5 + mariadb > 5.2:
+     * DROP TABLE FractionalSeconds;
+     * CREATE TABLE FractionalSeconds (time TIMESTAMP(3), value TIMESTAMP(3));
+     * INSERT INTO FractionalSeconds (time, value) VALUES( NOW(3), '1999-01-09 20:11:11.126' );
```

Comment:
More specific, some Databases will throw an error. Older MySql (mysql <= 5.5) is ignoring the fsp argument only while reading with *TIMESTAMP(3)*. 

If using it while creating a Table, most DB's and MySql <= 5.5 throws an error: 
*CREATE TABLE FractionalSeconds (time TIMESTAMP(3), value TIMESTAMP(3));* 
In older DB's you only can use: 
*CREATE TABLE FractionalSeconds (time TIMESTAMP, value TIMESTAMP);*